### DISCUSSION TITLE
Save song que list

### DISCUSSION THREAD

**Discussion by rysm83** [music-assistant/support#1858] [Feature requests and ideas]
I usually plays a particular song form Spotify and it auto populates a song que in Music Assistant.  I would like the ability to save that que.  I'm not sure if the que is generated at random based on the song I'm playing but this feature would be a nice to have so I can replace this again or add to a play list.

**Comment by OzGav** [comment]
Not 100% sure what you are asking. I assume you are using "Radio Mode" to create a queue based on your "particular song". The queue is generated by Spotify in your case. You cannot currently save the contents of a queue to a playlist.

**Comment by OzGav** [comment]
Duplicate https://github.com/orgs/music-assistant/discussions/1240
